index.php

```diff
@@ -34,6 +35,13 @@
         <input type="submit" value="Add to Collection"/>
     </form>
 
+    <h2>MADE A MISTAKE?</h2>
```

Comment:
If you're styling you might wanna put these in their own class so they don't act the same as all other text